pages/docs/tutorials/native/basic-kotlin-native-app.md

```diff
@@ -19,14 +19,14 @@ In this tutorial we'll see how to
 
 ## Obtaining the compiler
 
-Kotlin/Native is available for different macOS, Linux and Windows. Depending on the operating system we're working on, we'll need to download
+Kotlin/Native is available for macOS, Linux and Windows. Depending on the operating system we're working on, we'll need to download
 the correct compiler. While cross-platform compilation is possible (i.e. using one platform to compile for another), in this first tutorial
 we're going to compile for the same operating system we're running on. In our case this will be macOS. 
 
 We can obtain the latest version of the compiler from the [GitHub releases page](https://github.com/JetBrains/kotlin-native/releases).
 
 Once downloaded, we can uncompress it in any folder. For convenience it's useful to add the `bin` folder to the system path so that we can invoke the 
-compiler from any location.
+compiler from any location. 
```

Comment:
Clarify that the user needs to add the `bin` subdirectory of the unpacked directory? Also show example paths (unpack to `~/kotlin-native`, add `~/kotlin-native/bin` to PATH)?